gunrock/app/SimpleTemplate/SimpleTemplate_app.cu

```diff
@@ -0,0 +1,341 @@
+// ----------------------------------------------------------------------------
+// Gunrock -- Fast and Efficient GPU Graph Library
+// ----------------------------------------------------------------------------
+// This source code is distributed under the terms of LICENSE.TXT
+// in the root directory of this source distribution.
+// ----------------------------------------------------------------------------
+
+/**
+ * @file SimpleTemplate_app.cu
+ *
+ * @brief Simple Gunrock Application
+ */
+
+#include <gunrock/gunrock.h>
+#include <gunrock/util/test_utils.cuh>
+#include <gunrock/graphio/graphio.cuh>
+#include <gunrock/app/app_base.cuh>
+#include <gunrock/app/test_base.cuh>
+
+// <todo> change includes
+#include <gunrock/app/SimpleTemplate/SimpleTemplate_enactor.cuh>
+#include <gunrock/app/SimpleTemplate/SimpleTemplate_test.cuh>
+// </todo>
+
+namespace gunrock {
+namespace app {
+// <todo> change namespace
+namespace SimpleTemplate {
+// </todo>
```

Comment:
I really like the use `<todo>`, one note though is that using `TODO` in a comment in most text editors is actually highlighted in different colors, even in editors like vim it shows up really nice and bright. But maybe having it open and close with `<todo></todo>` tag helps.